jax/_src/scipy/linalg.py

```diff
@@ -1642,6 +1657,37 @@ def polar(a: ArrayLike, side: str = 'right', *, method: str = 'qdwh', eps: float
     ``posdef`` is either :math:`n \times n` or :math:`m \times m` depending on
     whether ``side`` is ``"right"`` or ``"left"``, respectively.
 
+  Example:
+
+    Polar decomposition of a 3x3 matrix:
+
+    >>> a = jnp.array([[1., 2., 3.],
+    ...                [5., 4., 2.],
+    ...                [3., 2., 1.]])
+    >>> U, P = jax.scipy.linalg.polar(a)
+
+    U is a Unitary Matrix:
+
+    >>> with jnp.printoptions(precision=2, suppress=True):
+    ...     print(U)
```

Comment:
Maybe print `U.T @ U` to show that the result is the identity?

Reply:
Thanks for the suggestion. Updated accordingly.